store/sqlstore/webhook_store.go

```diff
@@ -259,23 +269,29 @@ func (s SqlWebhookStore) GetOutgoingByChannel(channelId string, offset, limit in
 	return webhooks, nil
 }
 
-func (s SqlWebhookStore) GetOutgoingByTeam(teamId string, offset, limit int) ([]*model.OutgoingWebhook, *model.AppError) {
+func (s SqlWebhookStore) GetOutgoingByTeamByUser(teamId string, userId string, offset, limit int) ([]*model.OutgoingWebhook, *model.AppError) {
 	var webhooks []*model.OutgoingWebhook
 
-	query := ""
-	if limit < 0 || offset < 0 {
-		query = "SELECT * FROM OutgoingWebhooks WHERE TeamId = :TeamId AND DeleteAt = 0"
-	} else {
-		query = "SELECT * FROM OutgoingWebhooks WHERE TeamId = :TeamId AND DeleteAt = 0 LIMIT :Limit OFFSET :Offset"
+	query := "SELECT * FROM OutgoingWebhooks WHERE TeamId = :TeamId AND DeleteAt = 0"
```

Comment:
Not strictly necessarily for this PR, but would be nice to have it built with squirrel sql query builder.

Reply:
✅Done